Jenkinsfile

```diff
@@ -60,7 +60,7 @@ pipeline {
             steps {
                 container('java') {
                     script {
-                    sh "./gradlew test --no-daemon jacocoMergedReport shadowJar jib release ci \
+                    sh "./gradlew test --no-daemon jacocoMergedReport shadowJar jib release ci dockerPush\
```

Comment:
it think this needs docker to be available in the container , we could add it or move to anther step